product_attribute_types/README.rst

```diff
@@ -0,0 +1,35 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product attribute types
+=======================
+
```

Comment:
I can't find OCA guidelines, but as far I know, plural form is deprecated for module names